include/nn/util/util_BitFlagSet.h

```diff
@@ -202,13 +208,20 @@ struct BitFlagSet {
 
     template <int BitIndex>
     struct Flag {
-        static const int Index = BitIndex;
-        // todo: figure out how to make this not break past 63
-        static const BitFlagSet Mask = 1 << Index;
+        static_assert(BitIndex < N);
```

Comment:
should add a static_assert message (as message-less static_asserts are a C++17 feature)

Reply:
Will a copy paste of the check in string form work as I see that's being used elsewhere, also audio.h has some message-less static_asserts